### ISSUE TITLE
Installing Issue

### ISSUE THREAD

**Issue by o-enric** [Rounge-lab/VirMake#13]
Operating System: Linux horizon 5.15.0-91-generic
CPU: AMD Epyc 7763 processors
Memory: DDR4 RAM
Storage: SSD (Solid State Drive)
Graphics Accelerator: Tesla A40 GPU

Steps to Reproduce:

Activate the base environment:
conda activate base

Clone the VirMake repository:
git clone https://github.com/uio-bmi/VirMake.git

Navigate to the VirMake directory:
cd VirMake

Run the setup script:
python setup.py -y

Description:
Initially, I successfully installed Virmake, but it crashed at 61%. Now, I am unable to reinstall it.

Activating conda environment: .snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_
Traceback (most recent call last):
  File "/str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_/bin/DRAM-setup.py", line 5, in <module>
    from mag_annotator.database_processing import (prepare_databases, update_dram_forms,
  File "/str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_/lib/python3.10/site-packages/mag_annotator/database_processing.py", line 13, in <module>
    from skbio import read as read_sequence
  File "/str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_/lib/python3.10/site-packages/skbio/__init__.py", line 11, in <module>
    import skbio.io  # noqa
  File "/str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_/lib/python3.10/site-packages/skbio/io/__init__.py", line 248, in <module>
    import_module('skbio.io.format.lsmat')
  File "/str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_/lib/python3.10/site-packages/skbio/io/format/lsmat.py", line 77, in <module>
    from skbio.stats.distance import DissimilarityMatrix, DistanceMatrix
  File "/str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_/lib/python3.10/site-packages/skbio/stats/distance/__init__.py", line 197, in <module>
    from ._mantel import mantel, pwmantel
  File "/str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_/lib/python3.10/site-packages/skbio/stats/distance/_mantel.py", line 16, in <module>
    from scipy.stats import ConstantInputWarning
ImportError: cannot import name 'ConstantInputWarning' from 'scipy.stats' (/str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_/lib/python3.10/site-packages/scipy/stats/__init__.py)
[Thu Mar 28 02:53:01 2024]
Error in rule DRAMv:
    jobid: 2
    output: /str1/home/otavio.henrique/VirMake/databases/DRAM/DRAM_data
    conda-env: /str1/home/otavio.henrique/VirMake/workflow/.snakemake/conda/6c72fc1cbcaf2ae14237d0977e6eb871_
    shell:
        DRAM-setup.py prepare_databases --output_dir /str1/home/otavio.henrique/VirMake/databases/DRAM/DRAM_data --verbose --skip_uniref --threads 24;DRAM-setup.py export_config --output_file /str1/home/otavio.henrique/VirMake/databases/DRAM/DRAM_data/DRAM.config
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-03-28T025258.574203.snakemake.log
ERROR conda.cli.main_run:execute(124): `conda run snakemake --snakefile utils/setup_db.smk --cores 24 --configfile /str1/home/otavio.henrique/VirMake/workflow/config.yaml --use-conda --nolock --rerun-incomplete --directory /str1/home/otavio.henrique/VirMake/workflow` failed. (See above for error)

[setup.log](https://github.com/Rounge-lab/VirMake/files/14783454/setup.log)
[virsorter2_pass2.log](https://github.com/Rounge-lab/VirMake/files/14783455/virsorter2_pass2.log)

**Comment by torognes** [comment]
Hi!

Sorry for the late reply.

The problems you have experienced probably stems from conflicts in the environment used during setup of the DRAM databases. We have made many changes to VirMake recently, including the database setup. We have also tested the database setup and the rest of the pipeline from scratch, and the it seemed to work well.

I would therefore recommend that you try installing VirMake again from scratch.
